Makefile

```diff
@@ -1,49 +1,14 @@
-.PHONY: build run default debug test doc upstream downstream
+.PHONY: $(shell test -f common/common.mk || echo >&2 'Please do "git submodule update --init" first.')
 
-IMAGE_OPTIONS = \
-    -p 11211:11211
+BASE_IMAGE_NAME = memcached
+VERSIONS = 1.5
 
-VARIANT := upstream
-DISTRO = fedora-26-x86_64
-DG := /usr/bin/dg
-GOMD2MAN = /usr/bin/go-md2man
-
-DG_EXEC = ${DG} --max-passes 25 --distro ${DISTRO}.yaml --spec specs/configuration.yml --multispec specs/multispec.yml --multispec-selector variant=$(VARIANT)
-DISTRO_ID = $(shell ${DG_EXEC} --template "{{ config.os.id }}")
-
-IMAGE_REPOSITORY = $(shell ${DG_EXEC} --template "{{ spec.image_repository }}")
-
-default: run
-
-run: build
-	docker run -d $(IMAGE_REPOSITORY)
-
-debug: build
-	docker run -t -i $(IMAGE_OPTIONS) -e MEMCACHED_DEBUG_MODE $(IMAGE_REPOSITORY) bash
-
-build: doc
-	docker build --tag=$(IMAGE_REPOSITORY) -f Dockerfile.rendered .
+include common/common.mk
 
 test: build
```

Comment:
Please remove the `test` and `clean` rules as they are overriding the ones defined in common-container-scripts